reference/library/src/morpheus-master/sass/base/_extendables.scss

```diff
@@ -204,10 +204,22 @@
 	}
 }
 
-.button, button.ui-state-default{
+.button, button.ui-state-default, .btn.btn-default {
 	@include sakai_button( $button-background-color, $button-border-color, $button-text-color, $button-background-secondary-color );
 }
 
+.button.link, a.btn-link, button.btn-link {
+	@include sakai_button( #FFF, #FFF, $primary-color, #FFF );
```

Comment:
It's just for those who wants to modify their own skin, never say "always" in this community :D 

For now there's no variable for that background color but feel free to add one to the `_defaults.scss`, inherited from another or not